### ISSUE TITLE
Is the app dead?

### ISSUE THREAD

**Issue by gun4qmm7h** [AndreAle94/moneywallet#206]
I use moneywallet all the time, and notice that the app hasn't getting update. Is this project abandoned or is the updates really slow.

**Comment by 19Mahmoud19** [comment]
Yeah i think so, if u found any alternative with similar mechanism please mention it

**Comment by danielCamiloP** [comment]
The Dev mentions somewhere in the FAQ that he hasn't had a lot of time to continue development, sadly

**Comment by Vglith** [comment]
Does anyone know I suitable replacement? I've some backups from money wallet. So I hope I can use it!

**Comment by x-N0** [comment]
Easy as it gets to create an active OSS fork...

But it is not that easy to manage a FOSS project.

**Comment by davidegavio** [comment]
I have spoken with the dev, he said that he has little time to maintain it so if someone wants could become a maintainer in order to look after issues and prs
